anime_downloader/commands/dl.py

```diff
@@ -121,6 +123,34 @@ def command(ctx, anime_url, episode_range, url, player, skip_download, quality,
     if speed_limit:
         logger.info("Speed is being limited to {}".format(speed_limit))
     for episode in animes:
+        if video_info:
+            import cv2
+            cv2video = cv2.VideoCapture(episode.source().stream_url)
+            try:
+                if cv2video.isOpened():
+                    try:
+                        framecount = cv2video.get(cv2.CAP_PROP_FRAME_COUNT ) 
+                        frames_per_sec = cv2video.get(cv2.CAP_PROP_FPS)
+                        duration = framecount / frames_per_sec
+                        logger.info(f"Video duration (sec): {duration}")
+                    except:
```

Comment:
Perhaps capture the expected exceptions `as e` and use if statements for the logic instead of multiple try-except blocks?